### ISSUE TITLE
NGINX post installation of mod_pagespeed

### ISSUE THREAD

**Issue by mcpacific** [pagespeed/ngx_pagespeed#1409]
I originally installed mod_pagespeed on my cPanel server when I was using Apache alone. I have since installed NGINX. Do I need to transfer my configurations from pagespeed.conf to my nginx.conf file?

**Comment by oschaaf** [comment]
Are you using nginx to proxy to Apache? In that case I think you are fine.

**Comment by mcpacific** [comment]
Yes, as proxy. Very well. I'll keep everything in the pagespeed.conf file then. For some reason I can't seem to get the domain rewriting to work though... I would like to redirect all images to a different subdomain. I'm not sure if the filter order matters or if there are prerequisite requirements that I'm missing...

**Comment by oschaaf** [comment]
The filter order should not matter. A couple of questions:

- Have you seen / enabled the rewrite_domains filter?:
https://modpagespeed.com/doc/filter-domain-rewrite

- If you terminate https with nginx, you may need to take a look at `X-Forwarded-Proto` support:
https://modpagespeed.com/doc/https_support#RespectXForwardedProto
